### ISSUE TITLE
Sample Pages should link to their folders on GitHub

### ISSUE THREAD

**Issue by michael-hawker** [microsoft/WinUI-Gallery#413]
**Is this a totally new sample, an addition to an existing sample page, or a modification of an existing sample page?**
- [ ] New sample
- [ ] Addition to existing sample
- [x] Modification to existing sample

**If this is an addition/modification to an existing sample page, which one?**
All Sample Pages

**Additional context**
There's a lot of samples that have additional code that's not always shown on the page, or helper classes. It'd be great if the Documentation section also provided a link to GitHub for that sample; so that it's easier to access and view those, since that code isn't always visible in the application.

E.g. the VariedImageSizeLayout.cs link for ItemsRepeater

**Comment by stmoy** [comment]
We're closer to this via #392 but it sounds like even with that, there are pages that could use more links? Or do you think #392 gets us "close enough"?

**Comment by michael-hawker** [comment]
Thanks @stmoy, I must have missed the previous issue as it is closed now. #392 should be most of the way. I know for ItemsRepeater there are more 'helper' classes that are involved, so links to those would be useful. Maybe they are just links inline with the samples that they appear in? Not sure... 🤔

**Comment by chingucoding** [comment]
I don't think that currently, there are samples where any helper code is not inside the page's code. The only items needed for the samples, that are not on the sample page are a few mock sample pages, which we use to show navigating.

**Comment by chingucoding** [comment]
@michael-hawker Is there any specific controls besides ItemsRepeater where you would have expected more things to be linked?

**Comment by michael-hawker** [comment]
@chingucoding I think it's for any helper in samples that's not bubbled up. For instance ItemsRepeater uses the `ActivityFeedLayout` but that's not shown how to get the code for that example anywhere: https://github.com/microsoft/WinUI-Gallery/blob/winui2/WinUIGallery/Common/ActivityFeedLayout.cs
